src/routes/api.js

```diff
@@ -65,11 +74,37 @@ const removePlanInDB = async (req, res) => {
 };
 
 const removePlaceInDB = async (req, res) => {
-  res.json();
+  try {
+    const { id } = req.params;
+
+    await db.Plan.findByIdAndDelete(id);
+    const results = await db.Plan.find({});
+
+    res.json({
```

Comment:
Have a think about adding some specific status codes for these routes, such as 201 for created etc.